src/StringConverter.ino

```diff
@@ -565,7 +565,7 @@ void parseSystemVariables(String& s, boolean useURLencode)
   SMART_REPL_T(F("%sunrise"), replSunRiseTimeString)
 
   for (byte i = 0; i < CUSTOM_VARS_MAX; ++i) {
-    SMART_REPL(String(F("%customvar"))+toString(i+1,0)+String(F("%")), toString(customFloatVar[i],2))
+    SMART_REPL(String(F("%v"))+toString(i+1,0)+String(F("%")), toString(customFloatVar[i],2))
   }
 }
```

Comment:
At least place a check to see if it matches a `%v` as first characters or else skip this loop.
But even better is:
- check for `$v`
- if `sDigit(string[2])` do parse next 2 characters to a number
- call `SMART_REPL(String(F("%v"))+toString(number)+String(F("%")), toString(customFloatVar[number],2))`

What does this "2" mean in the last `toString` call? 2 decimals? Why 2?

Reply:
toString accepts a float and converts to a string.
The '2' are the number os decimals. I assumed that 2 decimals would be enough.
Should I increase them? To 4 maybe?

Reply:
max number of digits for a float is 7, so it's safe to set 6 decimals?

Reply:
Why not set the default output of a float? Afterall it will be changed into a float anyway.
And the Arduino implementation does trim trailing zeroes and probably check for some offset of epsilon to prevent values like 0.999999 to be printed instead of 1.0

Reply:
what is the function to convert from string to float natively in Arduino?
(I am still learning :)

Reply:
found it: String(float)